src/components/atoms/ChatMessageInfo/ChatMessageInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from "react";
+import classNames from "classnames";
+import dayjs from "dayjs";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+import { MessageToDisplay } from "types/chat";
+
+import { useProfileModalControls } from "hooks/useProfileModalControls";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import "./ChatMessageInfo.scss";
+
+export interface ChatMessageInfoProps {
+  message: MessageToDisplay;
+  deleteMessage: () => void;
+  isReversed?: boolean;
+}
+
+export const ChatMessageInfo: React.FC<ChatMessageInfoProps> = ({
+  message,
+  deleteMessage,
+  isReversed = false,
+}) => {
+  const { ts_utc, author, canBeDeleted } = message;
+  const { openUserProfileModal } = useProfileModalControls();
+
+  const timestamp = ts_utc.toMillis();
+
+  const openAuthorProfile = useCallback(() => {
+    openUserProfileModal(author);
+  }, [openUserProfileModal, author]);
+
+  const containerClasses = classNames("ChatMessageInfo", {
+    "ChatMessageInfo--reverse": isReversed,
+  });
+
+  return (
+    <div className={containerClasses} onClick={openAuthorProfile}>
+      <UserAvatar user={author} />
+      <span className="ChatMessageInfo__author">{author.partyName}</span>
+      <span className="ChatMessageInfo__time">
+        {dayjs(timestamp).format("h:mm A")}
```

Comment:
Changed the lib in https://github.com/sparkletown/sparkle/pull/1374/commits/7387c8c75601fb1b7713d3b33fc975f91e88a9aa

I wonder if there is sense moving this into utils/time, since it's just a direct method call?

Reply:
It seems like a common format that we would want to re-use in other places in the app in future, so would make sense to me to make a small wrapping function for it in `utils/time` with an appropriate name.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1374/commits/1c37309810e063a5f6a570371d22071088c1898d